tests/src/FunctionalJavascript/CustomFieldSubmissionTest.php

```diff
@@ -219,15 +219,15 @@ public function testSubmitWebform() {
     $this->enableCheckboxOnElement('edit-webform-ui-elements-civicrm-1-contact-1-cg1-custom-4-operations');
 
     // ToDo: Enable Static Option and Edit Label
-    // $checkbox_edit_button = $this->assertSession()->elementExists('css', '[data-drupal-selector="edit-webform-ui-elements-civicrm-1-contact-1-cg1-custom-3-operations"] a.webform-ajax-link');
-    // $checkbox_edit_button->click();
-    // $this->assertSession()->assertWaitOnAjaxRequest();
-    // $this->htmlOutput();
-    // $this->getSession()->getPage()->selectFieldOption("properties[civicrm_live_options]", 0);
-    // $this->assertSession()->assertWaitOnAjaxRequest();
-    // $this->assertSession()->waitForField('properties[options][options][civicrm_option_1][label]');
-    // $this->getSession()->getPage()->fillField('properties[options][options][civicrm_option_1][label]', 'Red - Recommended');
-    // $this->htmlOutput();
+    $checkbox_edit_button = $this->assertSession()->elementExists('css', '[data-drupal-selector="edit-webform-ui-elements-civicrm-1-contact-1-cg1-custom-3-operations"] a.webform-ajax-link');
+    $checkbox_edit_button->click();
+    $this->assertSession()->assertWaitOnAjaxRequest();
+    $this->htmlOutput();
+    $this->getSession()->getPage()->selectFieldOption("properties[civicrm_live_options]", 0);
+    $this->assertSession()->assertWaitOnAjaxRequest();
+    $this->assertSession()->waitForField('properties[options][options][civicrm_option_1][label]');
+    $this->getSession()->getPage()->fillField('properties[options][options][civicrm_option_1][label]', 'Red - Recommended');
+    $this->htmlOutput();
```

Comment:
We could put `$this->createScreenshot($this->htmlOutputDirectory . '/test1.png');` before and test1 after changing live options